app/helpers/abr_pdf_methods.rb

```diff
@@ -115,13 +115,25 @@ def pdf_template_path
     Rails.root.join("data/abr_pdfs/#{abbrev}/#{pdf_template_name}")
   end
 
+  def pdf_template_with_cover_filename
+    pdf_template_name.gsub(/\.pdf\z/, '_with_cover.pdf')
+  end
+
+  def pdf_template_with_cover_path
+    return nil unless has_pdf_template?
+    abbrev = home_state_abbrev.downcase
+    Rails.root.join("data/abr_pdfs/#{abbrev}/#{pdf_template_with_cover_filename}")
+  end
+
   def pdf_writer
     return nil unless has_pdf_template?
     if @pdf_writer.nil?
       @pdf_writer = PdfAbrWriter.new
       @pdf_writer.assign_attributes(self.to_pdf_hash)
+      @pdf_writer.pdf_cover_values = make_cover_fields_hash 
       
       @pdf_writer.pdf_template_path = pdf_template_path
+      @pdf_writer.pdf_template_with_cover_path = pdf_template_with_cover_path
       @pdf_writer.delivery_address = state_registrar_address
```

Comment:
I think this isn't used anymore so can be removed?

Reply:
I think you mean the line below... the delivery address. 

We're not creating the HTML cover page anymore, so yes that can be removed.

Reply:
...and all the related functions.